--- lirccd-0.9.1/lib/lirc_client.c.orig	2004-03-13 14:53:47.000000000 +0100
+++ lirccd-0.9.1/lib/lirc_client.c	2013-02-02 18:24:56.229075109 +0100
@@ -277,7 +277,7 @@
     }
 }
 
-static char *getline(char *type, int block)
+static char *lirccd_getline(char *type, int block)
 {
     int i, o, t;
     char *n, *buf;
@@ -382,7 +382,7 @@
     char **w, *l;
     int i, ret;
     
-    if((l = getline("repl", block)) == NULL)
+    if((l = lirccd_getline("repl", block)) == NULL)
 	return(-1);
     w = wordexp(l);
     free(l);
@@ -410,7 +410,7 @@
 {
     char **w, **wp, *l;
     
-    if((l = getline("mesg", block)) == NULL)
+    if((l = lirccd_getline("mesg", block)) == NULL)
     {
 	if(mesg != NULL)
 	    *mesg = NULL;
@@ -439,7 +439,7 @@
 {
     char **w, **wp, *l;
     
-    if((l = getline("bttn", block)) == NULL)
+    if((l = lirccd_getline("bttn", block)) == NULL)
     {
 	errno = EAGAIN;
 	return(-1);
